### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by hddn**
Shopping cart and order processing added.

### CODE REVIEW

File: cart/urls.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from django.conf.urls import url
+from views import add_to_cart, remove_from_cart, show_cart, clear_all
```
Comment (cosprm) [inline cart/urls.py]: Exception Value: No module named 'views'
use cart.views

Comment (hddn) [reply cart/urls.py]: It is because of python 3, I guess.

File: cats_shop/views.py [hunk 2]
```diff
@@ -10,3 +16,34 @@ class CatsList(ListView):
...
+            if form.is_valid():
+                order.save()
+            cart = Cart(request.session)
+            for item in cart.items:
+                order_item = OrderPosition()
+                order_item.order = order
+                order_item.product = item.product
+                order_item.save()
+            cart.clear()
+            return super(OrderAddView, self).post(request, *args, **kwargs)
```
Comment (VenomBrend) [inline cats_shop/views.py:49]: I don't know "why", but it creates two records in my table Order for one order
http://prnt.sc/cm2r48

Comment (hddn) [reply cats_shop/views.py:49]: Fixed

**Pull request merged**